### ISSUE TITLE
Studio object viewer plugin font

### ISSUE THREAD

**Issue by ryzom-pipeline** [ryzom/ryzomcore#199]
**[Original report](https://bitbucket.org/ryzom/ryzomcore/issue/199) by Meelis Mägi (Bitbucket: [Meelis Mägi](https://bitbucket.org/Meelis Mägi), ).**

----------------------------------------

Font is loaded without looking up it's full path first.

With this patch, plugin will still crash if font is not found, but at least has a higher chance of running when font is in CPath search
```
#!c++

--- a/code/studio/src/plugins/object_viewer/object_viewer.cpp
+++ b/code/studio/src/plugins/object_viewer/object_viewer.cpp
@@ -115,7 +115,7 @@ void CObjectViewer::init( NL3D::UDriver

        // create the text context
        nlassert(!_TextContext);
-       _TextContext = _Driver->createTextContext(_FontName);
+       _TextContext = _Driver->createTextContext(NLMISC::CPath::lookup(_FontName));
        nlassert(_TextContext);

        NL3D::CBloomEffect::instance().setDriver(_Driver);

```

**Comment by ryzom-pipeline** [comment]
**Original comment by Jan Boon (Bitbucket: [Jan Boon](https://bitbucket.org/Jan Boon), ).**

----------------------------------------

Looks good

**Comment by ryzom-pipeline** [comment]
**Original comment by Jan Boon (Bitbucket: [Jan Boon](https://bitbucket.org/Jan Boon), ).**

----------------------------------------

Fix #199 Studio object viewer plugin font
